### PR TITLE
[Fixes #35] Migrated existing doc to docusaurus

### LINKED ISSUE #35: Migrate existing documentation
Specifically:

* https://docs.google.com/document/d/1ixyYDKKz_cMJd3TaeBLXLGKiUJRkGlq1AWTkm2dYYfk/edit
* https://docs.google.com/document/d/1-8Wctqqsh1OSaDu6BKfwBGc5rhJFsGdwYUpHtGLbzhg/edit

### PR DESCRIPTION
**Pull request by jeluard**

### CODE REVIEW

File: docs/Manifesto.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+Extensions are a way for 3rd party developers to grow the Status application while reusing its features.
+Lightweight, they provide native feeling and have access to the full ethereum decentralized experience augmented by Status unique features.
+
+## Goal
+
+Status is investing lots of energy to make decentralized primitives available on mobile platforms. Let’s allow ethereum developers to focus on what they do best: innovate. They can leverage Status first class features and UX.
+Open and free, Status is now extensible. 
+
+Specifically we want:
+* To make Status awesome for end users who want to easily use Mini DApps
```
Comment (flexsurfer) [inline docs/Manifesto.md]: Mini DApps ?

File: docs/Manifesto.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## How open?
+
+Status is inherently malleable. It is Status’s responsibility to carefully choose what is available to extensions and how it can be extended.
+
+Openness is fundamental to Status. It is based on three pillars: data access, API and extension points.
+There is a delicate balance between provide access needed to create awesome DApps while ensuring strong data control to users. To that end, a new security model that emphasizes data destination will be considered (in p2p world, the knowledge of what you send and who you send it to is fundamental). 
+
+### Data access
+
+Data in status in kept is kept in a single, tree-based entity. Extensions will rely on a query mechanism for accessing subset of this data. Depending on the security granted, extensions might have access to most of the data manipulated by status.
```
Comment (flexsurfer) [inline docs/Manifesto.md]: `in kept is kept` typo?

File: docs/Manifesto.md (hunk 1 above)
Comment (jeluard) [reply docs/Manifesto.md]: Or native DApps

File: docs/Manifesto.md (hunk 2 above)
Comment (jeluard) [reply docs/Manifesto.md]: :sweat_smile:

**Pull request merged**